OpenUpSA/wazimap-ng-ui
# Hiding subindicators in data mapper where no data is available

## Description
Checking if a sub-indicator has data to show and not adding it in data mapper and the rich data panel if no data is available

## Related Issue
https://github.com/OpenUpSA/wazimap-ng-ui/issues/216

## How to test it locally
Look at the `geo:366005001` on YE. you should see no empty sections. 

https://deploy-preview-242--wazimap-staging.netlify.app/#geo:366005001

## Screenshots
Before
![image](https://user-images.githubusercontent.com/53019884/110659150-5f301000-81d3-11eb-95da-5272fc127100.png)

After
![image](https://user-images.githubusercontent.com/53019884/110659188-66efb480-81d3-11eb-8e27-e7408e8fc9ad.png)


## Changelog

### Added

### Updated
* `utils.js` to add the function that checks if a subcategory has children
* `profile/category.js` to hide the subcategories with no children in the rich data panel
* `elements/menu.js` to hide the subcategories with no children in the data mapper

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
